sys/include/net/ethertype.h

```diff
@@ -34,7 +34,8 @@ extern "C" {
 #define ETHERTYPE_RESERVED      (0x0000)    /**< Reserved */
 #define ETHERTYPE_IPV4          (0x0800)    /**< Internet protocol version 4 */
 #define ETHERTYPE_ARP           (0x0806)    /**< Address resolution protocol */
-#define ETHERTYPE_NDN           (0x0801)    /**< Parc CCNX */
```

Comment:
Could this change have an impact on other named data network (CCN-lite) implementations ?

Reply:
I don't think so. CCN-lite for RIOT uses only NDN for now

Reply:
Actually, this should be changed to 
```
#define ETHERTYPE_NDN 0x8624
```

This is the value we are using in other places.  This, probably, was just a quick hack.

Reply:
I'm slightly confused by the PARC CCNX comment.

Reply:
I just tested the ccn-lite-relay example, it behave the same as on master.

Reply:
actually this line is the one deleted, ETHERTYPE_NDN is defined as 0x8624 in this PR.
but ETHERTYPE_CCNX is still defined as 0x0801. What do we do about these lines ?